omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/FieldTypeResource1_8.java

```diff
@@ -29,6 +32,25 @@
         "1.8.*", "1.9.*", "1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*" })
 public class FieldTypeResource1_8 extends MetadataDelegatingCrudResource<FieldType> {
 	
+	public Model getGETModel(Representation rep) {
+		ModelImpl modelImpl = (ModelImpl) super.getGETModel(rep);
+		if (rep instanceof DefaultRepresentation || rep instanceof FullRepresentation) {
+			modelImpl
+			        .property("isSet", new BooleanProperty()._default(false));
+		}
+		return modelImpl;
+	}
+	
+	@Override
+	public Model getCREATEModel(Representation rep) {
+		return super.getCREATEModel(rep); //FIXME missing props
```

Comment:
Yes. As discussed with Shreyans they will be fixed in a separate issue. These will require minor to no change at all. But may need testing and discussion.